CHANGES.txt

```diff
@@ -11,6 +11,7 @@ NOTE: Please include a reference to any Issues resolved by your changes in the b
 RELEASE  VERSION/DATE TO BE FILLED IN LATER
 
   From Daniel Moody:
+    - Add no_progress (-Q) option as a set-able option
```

Comment:
Add bullet here with example and a note that the initial 

> scons: Reading SConscript files ...

Will still be output?

Reply:
yes it will, updated the notes.